### ISSUE TITLE
Largenode tidbits

### ISSUE THREAD

**Issue by vortexing** [FredHutch/wiki#365]
**Proposed Domain**
https://sciwiki.fredhutch.org/scicomputing/compute_jobs/

**Content Summary**
Discuss more about how to use largenode.  Looks like you HAVE to ask for memory, and the minimum memory you need to ask for to get it to go is 42+GB?  Perhaps add a largenode example?  Also, maybe an example in a bash script header rather than a one liner?  
```
#!/bin/bash
#SBATCH --output=Logs/%x_%A_%a.out
#SBATCH -p largenode
#SBATCH -c 6
#SBATCH --mem=43G
```  

**Local Content Expert(s)**
SciComp crew

**Comment by bdconnolly** [comment]
@vortexing I have added a example about submitting a job to largenode partition to https://sciwiki.fredhutch.org/scicomputing/compute_jobs/#examples

Is this what you were looking for?

**Comment by bdconnolly** [comment]
This issue has been fixed in merge request #422
